session6/com.itschool.course/src/homework/Ex19.java

```diff
@@ -0,0 +1,18 @@
+//package homework;
+//
+//import java.util.Scanner;
+//public class Ex19 {
+//
+//    public static void main(String[] args) {
+//
+//        //help here, please
+//
+////        Scanner scanner = new Scanner(System.in);
+////        System.out.println("Enter a number reprezenting the age, please.");
+////
+////        int age = scanner.nextInt();
+////        String result = (age < 12) ? "Child" : (age >=12 && age <=17) ? "Teenager" : (age >=18 && age<=64) ? "Adult" : (age >= 65) ? "Senior";
```

Comment:
here:
`String result = (age < 12) ? "Child" : (age >=12 && age <=17) ? "Teenager" : (age >=18 && age<=64) ? "Adult" : "Senior";`
But I know this is a pain! This is actually the purpose of this exercise - to see how difficult is to it and read it with ternary operator. The correct approach here would be to use `if-else if` blocks. Ternary operator should be used only for simple comparison with only 3 expressions involved.